support/modules/single-file

```diff
@@ -53,8 +64,7 @@ case "$STATE" in
         dest_dir=$(cat "${dest_dir_file}")
         test -z "$dest_dir" -o -z "$filename" && \
             echo "Fatal error: dest_dir or filename are undefined." && exit 1
-        rm "${dest_dir}/${filename}"
```

Comment:
Question:
How does removing this relate to `safe_copy` of the files?

Reply:
Removing it first means that it's not atomic. There is a window in time where the file does not exist. This is why we use `mv` inside `safe_copy` as the update step, because it is atomic.